### ISSUE TITLE
MechJeb autopilot not found when MJ part is on vessel

### ISSUE THREAD

**Issue by mathuin** [Mihara/RasterPropMonitor#651]
I have `MechJeb2 2.6.1.0` installed in my KSP instance along with `RasterPropMonitor 1`:v0.29.2` and RasterPropMonitor-Core 1:v0.29.2`.  

When I hit the fourth button on the top row, I expect to see a menu for the MJ autopilot.  Instead, it says "MechJeb Autopilot not found."

If you need logs, please let me know and I can post them!

**Comment by MOARdV** [comment]
I infer from your mention of `RasterPropMonitor 1` and `RasterPropMonitor-Core 1` that you are using CKAN to install and manage your mods.  CKAN installation is not supported.  If you can reproduce the problem with a manual installation of MechJeb and RPM, please provide logs.

**Comment by mathuin** [comment]
I moved aside my installation and had Steam freshly install KSP 1.3.1.1891.  I installed MechJeb from Curse because it had the 2.6.1.0 release and its GitHub release page only showed 2.6.0.  I installed RPM from its GitHub release page because it had the 0.29.2 release and Curse only had 0.18.3.  I can confirm that I see three directories under `GameData/JSI` -- `Agencies`, `RasterPropMonitor`, and `RPMPodPatches.

I created a sandbox game and opened the Kerbal X rocket in the VAB.  I attached a MechJeb part to the command pod, saved the vessel under a different name, and sent it to the launch pad.  When I looked at the pod from Jeb's perspective, I saw no monitors at all!

The log file is 3341 lines long.  Would you like the whole log, or specific sections?

Here's one which shows I installed RPM correctly:

```
************************************************************************

Environment Info
Unix 7FFFFFFFFFFFFFFF  Args: KSP.x86_64

Mod DLLs found:
Stock assembly: Assembly-CSharp v0.0.0.0
RasterPropMonitor v0.29.2.13391
MechJeb2 v2.5.1.0 / v / v2.6.1.0
Stock assembly: KSPSteamCtrlr v0.0.1.35
Stock assembly: Steamworks.NET v9.0.0.0 / v9.0.0

Folders and files in GameData:
JSI
MechJeb2
Stock folder: Squad


************************************************************************
```

**Comment by MOARdV** [comment]
Ah.  You are missing Module Manager.  MM is required to update the interiors of the vessels.  Sorry I didn't mention that.

**Comment by mathuin** [comment]
I downloaded `ModuleManager.3.0.1.dll` from the links posted in the MM forum thread and copied it into `GameData/`.  MM reported 54 patches applied on startup.  Started up my game, brought out my Kerbal X with MechJeb, and now screens are present and the first click on the fourth button brings up autopilot control (ascent guidance, land somewhere, etc).  So good!

Back to my original game.  I used CKAN to uninstall MechJeb2 and both RasterPropMonitor mods.  This uninstalled Vessel View and its RPM mod as well as Probe Control Room Recontrolled.  I then used tar to copy over the JSI and MechJeb2 folders (MM 3.0.1 was already there).  I then sent a rocket to the launchpad, confirmed that it had a MJ part on it, and checked the IVA. Screens still present, autopilot control still present.  So either the installation via CKAN was at fault or the other mods that were removed.

Installed PCRC 1.2.2.9 and VVRPM 0.8.6 via zip files.  For the latter, I chose 'No' when asked to modify existing RPM files.  Screens still present, autopilot control still present.  I now think that VVRPM's inclusion of RPM files is the root cause of my issue, but it is definitely not your problem.
